packages/mds-jurisdiction-service/tests/index.spec.ts

```diff
@@ -28,7 +30,21 @@ const LAST_WEEK = TODAY - days(7)
 
 const JurisdictionServer = JurisdictionServiceManager.controller()
 
-describe('Write/Read Jurisdictions', () => {
+describe('Test Migrations', () => {
+  it('Run Migrations', async () => {
+    const connection = await createConnection(ormconfig as ConnectionOptions)
+    await connection.runMigrations()
```

Comment:
Is the point of this test to ensure that migrations run without throwing? I'm not totally sure what's being tested here.

Reply:
Yes. To ensure that running and reverting don't throw.